### ISSUE TITLE
gisticChromPlot - hg38 issue

### ISSUE THREAD

**Issue by t-neumann** [PoisonAlien/maftools#365]
Hi,

I am using the `gisticChromPlot` function on `hg38` variants.

What I am doing is this:

```
> gisticChromPlot(gistic = MCC.gistic, markBands = "all", ref.build = "hg38")
Error in if (!is_overlap(x1 - 0.5 * wid, y1 - 0.5 * ht, wid, ht, boxes) &&  : 
  missing value where TRUE/FALSE needed
```

Now when I use `hg19` as `ref.build` it works - any idea what could be going on here?

```
> gisticChromPlot(gistic = MCC.gistic, markBands = "all", ref.build = "hg19")
> 
```

Thanks!

**Comment by PoisonAlien** [comment]
Its hard to guess. Is it possible to get a reproducible example?

**Comment by t-neumann** [comment]
@PoisonAlien Can I send you my blinded GISTIC output files via mail?

This is the commands I used:

```
dummy.gistic = readGistic(gisticAllLesionsFile = all.lesions,
                        gisticAmpGenesFile = amp.genes,
                        gisticDelGenesFile = del.genes,
                        gisticScoresFile = scores.gis
)

gisticChromPlot(gistic = dummy.gistic, markBands = "all", ref.build = "hg38")
```

**Comment by PoisonAlien** [comment]
Sure, I understand the concern. You can mail me at anandmt3@gmail.com

**Comment by t-neumann** [comment]
@PoisonAlien Thanks a lot, that indeed fixed it!

Btw on an unrelated topic - you have this built in `prepareMutSig` function in `maftools`. I'm curious, does it prepare any input files ready to be run with `mutSigCV`? The documentation is rather minimal on this, so I cannot really tell. Do you have any experience in running `mutSigCV` - from what I read I did not find any `hg38` support.

**Comment by PoisonAlien** [comment]
Its been a while since I used mutSig. However, there was a news recently that Broad released a newer updated version of MutSig. You will have to search - they sometime don't update their [cga](https://software.broadinstitute.org/cancer/cga/)

`prepareMutSig` just fixes the gene names which becomes compatible with MutSig. Some of the hugo symbols are not supported by MutSig and they are generally ignored by the program. I opened an [issue](https://software.broadinstitute.org/cancer/cga/node/716) on their forum a while back but I did not get any response - hence this function to fix it.
